openedx/core/djangoapps/course_apps/tests/test_api.py

```diff
@@ -0,0 +1,51 @@
+"""
+Tests for the python api for course apps.
+"""
+from unittest import mock
+from unittest.mock import Mock
+
+import ddt
+from django.test import TestCase
+from opaque_keys.edx.locator import CourseLocator
+
+from .utils import make_test_course_app
+from ..api import is_course_app_enabled, set_course_app_enabled
+
+
+@ddt.ddt
+@mock.patch("openedx.core.djangoapps.course_apps.api.CourseAppsPluginManager.get_plugin")
+class CourseAppsPythonAPITest(TestCase):
+    """
+    Tests for the python api for course apps.
+    """
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.course_key = CourseLocator(org="org", course="course", run="run")
+        self.default_app_id = "test-app"
+
+    @ddt.data(True, False)
+    def test_plugin_enabled(self, enabled, get_plugin):
+        """
+        Test that the is_enabled value is used in case that method exists.
```

Comment:
Test description needs to be updated now that we do not check for the existence of the method